tests/integration/web/machinetracker_test.py

```diff
@@ -1,8 +1,11 @@
 from datetime import datetime
 
+from django.urls import reverse
+from django.utils.encoding import smart_str
+
 from nav.models.fields import INFINITY
 from nav.models.manage import Arp
-from nav.web.machinetracker.views import get_netbios_query
+from nav.web.machinetracker.views import get_netbios_query, VALID_HELP_TAB_NAMES
```

Comment:
```suggestion
from nav.web.machinetracker.views import VALID_HELP_TAB_NAMES, get_netbios_query
```

If you're ever uncertain about import order (as I am often), you can simply use ruff as help as following: `ruff check --select="I" filepath`

Reply:
Anyways, done :)